frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -245,8 +255,10 @@ export const EditZing = () => {
           <StyledLogo />
           <StyledText>{courseInfo.names.join(', ')}</StyledText>
         </StyledLogoWrapper>
-        s
-        <Button sx={{ height: '40px', mt: '10px' }}>
+        <Button
+          sx={{ height: '40px', mt: '10px' }}
+          onClick={() => setIsEmailing(!isEmailing)}
```

Comment:
Entrypoint 2 for connecting to new button + edit

Reply:
Merge from main into ur branch and update with the new code here